src/graph/util/ExpressionUtils.cpp

```diff
@@ -1058,6 +1061,20 @@ bool ExpressionUtils::isGeoIndexAcceleratedPredicate(const Expression *expr) {
   return false;
 }
 
+Status ExpressionUtils::extractPropsFromExprs(const Expression *expr,
+                                              PropertyTracker &propsUsed,
+                                              const graph::QueryContext *qctx,
+                                              GraphSpaceID spaceID,
+                                              const std::string &entityAlias) {
```

Comment:
Define a more generic interface for utility functions.

Reply:
I don't get it. Could you have a explain?